common/prque/prque_byteSlice_test.go

```diff
@@ -0,0 +1,92 @@
+// CookieJar - A contestant's algorithm toolbox
+// Copyright (c) 2013 Peter Szilagyi. All rights reserved.
+//
+// CookieJar is dual licensed: use of this source code is governed by a BSD
+// license that can be found in the LICENSE file. Alternatively, the CookieJar
+// toolbox may be used in accordance with the terms and conditions contained
+// in a signed written agreement between you and the author(s).
+
+package prque
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/klaytn/klaytn/common/math"
+
+	"github.com/klaytn/klaytn/common"
+)
+
+func TestPrqueByteSlice(t *testing.T) {
```

Comment:
Is it okay skipping a test for `NewByteSliceInverted`?

Reply:
I added the testcase.